packages/special-pages/pages/new-tab/app/settings.js

```diff
@@ -0,0 +1,46 @@
+export class Settings {
+    /**
+     * @param {object} params
+     * @param {{name: ImportMeta['platform']}} [params.platform]
+     */
+    constructor ({
+        platform = { name: 'windows' }
+    }) {
+        this.platform = platform
+    }
+
+    /**
+     * @param {keyof import("../../../types/duckplayer").DuckPlayerPageSettings} named
+     * @param {{state: 'enabled' | 'disabled'} | null | undefined} settings
+     * @return {Settings}
+     */
+    withFeatureState (named, settings) {
```

Comment:
A leftover method from DuckPlayer settings that can be removed?

Reply:
yep, removing